### DISCUSSION TITLE
Hi I started a new Nocodb server by [Jul 06]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2676] [Discord Threads]
> 🙋‍♂️ **stornk**

**Comment by o1lab** [comment]
> 🙋‍♂️ **stornk**  
#### well, I just went ahead with the 1-click Heroku deployment and that does seem to work...  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/993823887102058566)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
